README.md

```diff
@@ -57,7 +57,7 @@ brew install grpcurl
 ### From Source
 You can use the `go` tool to install `grpcurl`:
 ```shell
-go get github.com/fullstorydev/grpcurl
+go get -u github.com/fullstorydev/grpcurl
```

Comment:
Perhaps this should instead be a comment, suggesting people to use the `-u` flag for `go get` if they are not yet using Go modules?